CHANGELOG.md

```diff
@@ -4,6 +4,9 @@ This project adheres to [Semantic Versioning](http://semver.org/).
 This CHANGELOG follows the format listed [here](https://github.com/sensu-plugins/community/blob/master/HOW_WE_CHANGELOG.md)
 
 ## [Unreleased]
+### Changed
+- `check-http.rb`: Add options to set open-timeout and read-timeout for Net:HTTP. Improve output on what Net::HTTP timeout was encountered.
```

Comment:
ideally I'd like to see the option names being called out here so someone reading this knows exactly what to update to leverage. 

For example:
```
## [Unreleased]
### Added
- `check-http.rb`: Add options to set `--open-timeout` and `--read-timeout` for Net:HTTP. Additionally rescue `Net::OpenTimeout` and `Net::ReadTimeout` exception classes (@mygithubuser)
- `check-http.rb`: exposed `--dns-timeout` and rescue on `Resolv::ResolvTimeout` exception class (@mygithubuser)

### Changed
- `check-http.rb`: switched to using rubies DNS resolver (which looks at `/etc/resolv.conf`) and rescue `Resolv::ResolvError` exception class for generic DNS failures (@mygithubuser)
```